test/test_yahoo.py

```diff
@@ -6,13 +6,21 @@
 else:
     from unittest import main as test_main, SkipTest, TestCase
 
-from yahoo_finance import Currency, Share, edt_to_utc, get_date_range
-
+from yahoo_finance import Currency, Share, edt_to_utc, get_date_range,\
+    SymbolDoesNotExistError
 
 class TestShare(TestCase):
 
     def setUp(self):
-        self.yahoo = Share('YHOO')
+        self.symbol = 'YHOO'
+        self.yahoo = Share(self.symbol)
+
+    def test_nonexisting_symbol(self):
+        nonexisting = 'NOTEXISTING'
```

Comment:
make this a staic variable since it's reused at line 101?

e.g., add this at line 17

```
NON_EXISTING = 'NOTEXISTING'
```

and then replace 21 and 101 with
```
share = Share(TestShare.NON_EXISTING)
```